Content.Client/Chat/Managers/ChatManager.cs

```diff
@@ -75,6 +75,10 @@ public void SendMessage(ReadOnlyMemory<char> text, ChatSelectChannel channel)
                     _consoleHost.ExecuteCommand($"tsay \"{CommandParsing.Escape(str)}\"");
                     break;
 
```

Comment:
Whenever we edit preexisting systems, we should do what Corvax does, mark the start and end of it with comments. Makes it so nice to go back and look at, as well as for anyone else exploring our code.
Even minor stuff like this, it means you can search for, say, start of SimpleStation14 in content and find anything we've fiddled with.

```suggestion

                    // Start of SimpleStation14 Shadowkin chat
```